core/src/main/java/jenkins/util/TreeString.java

```diff
@@ -43,8 +43,6 @@
  * @author Kohsuke Kawaguchi
  * @since 1.473
  */
-// CHECKSTYLE:OFF
```

Comment:
This file complies with our Checkstyle checks, so I see no reason to turn Checkstyle off.